src/about-us/our-mission-values-and-principles.md

```diff
@@ -124,15 +124,14 @@ However, there are some things that we must keep private.
 
 ### Make everything we do accessible
 
-At dxw we want to make sure that everyone can access the digital services we
+At dxw we want to make sure that everyone can access the services we
```

Comment:
why not products and services?
why just services?

Reply:
Because we are usually talking about public services or government services. Some clients do use the word 'product' to describe the large parts that make up a service. But nobody talks about 'public products'. And GDS call Notify, Pay, Forms and One Login 'platforms' or 'tools' or 'technologies' rather than products.

Reply:
thank you for explaining this